src/tests/functional/plugin/cpu/single_layer_tests/eltwise.cpp

```diff
@@ -270,21 +286,37 @@ const auto params_4D_fusing = ::testing::Combine(
 
 INSTANTIATE_TEST_SUITE_P(smoke_CompareWithRefs_4D_Fusing, EltwiseLayerCPUTest, params_4D_fusing, EltwiseLayerCPUTest::getTestCaseName);
 
-const auto params_4D_emptyCPUSpec = ::testing::Combine(
+const auto params_4D_FP32_emptyCPUSpec = ::testing::Combine(
         ::testing::Combine(
                 ::testing::ValuesIn(static_shapes_to_test_representation(inShapes_4D)),
                 ::testing::ValuesIn(eltwiseOpTypesDiffInp),
                 ::testing::ValuesIn(secondaryInputTypes),
                 ::testing::ValuesIn(opTypes),
-                ::testing::ValuesIn(netType),
+                ::testing::Values(ElementType::f32),
                 ::testing::Values(ElementType::f32),
                 ::testing::Values(ElementType::f32),
                 ::testing::Values(CommonTestUtils::DEVICE_CPU),
                 ::testing::Values(additional_config)),
         ::testing::Values(emptyCPUSpec),
         ::testing::Values(emptyFusingSpec));
 
-INSTANTIATE_TEST_SUITE_P(smoke_CompareWithRefs_4D_emptyCPUSpec, EltwiseLayerCPUTest, params_4D_emptyCPUSpec, EltwiseLayerCPUTest::getTestCaseName);
+INSTANTIATE_TEST_SUITE_P(smoke_CompareWithRefs_4D_FP32_emptyCPUSpec, EltwiseLayerCPUTest, params_4D_FP32_emptyCPUSpec, EltwiseLayerCPUTest::getTestCaseName);
+
+const auto params_4D_BF16_emptyCPUSpec = ::testing::Combine(
```

Comment:
To be honest I don't think we need to extend each test suite with BF16 prec. since it increases number of test cases to much. Lets have separate small suite that checks only BF16 for avx2/sse42 only.

Reply:
BTW it's remainder after my attempt to add removing of limitation after inputs in this PR on March. At the moment it's not necessary change while